tests/e2e/specs/backend/active-filters.test.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import {
+	getAllBlocks,
+	switchUserToAdmin,
+	getEditedPostContent,
+} from '@wordpress/e2e-test-utils';
+import { clearAndFillInput } from '@woocommerce/e2e-utils';
+import { insertBlockDontWaitForInsertClose } from '../../utils.js';
+import { visitBlockPage } from '@woocommerce/blocks-test-utils';
+
+const block = {
+	name: 'Active Product Filters',
+	slug: 'woocommerce/active-filters',
+	class: '.wc-block-active-filters',
+};
+
+describe( `${ block.name } Block`, () => {
+	beforeAll( async () => {
+		await switchUserToAdmin();
+		await visitBlockPage( `${ block.name } Block` );
+	} );
+
+	it( 'can be inserted once', async () => {
+		await insertBlockDontWaitForInsertClose( block.name );
+		expect( await getAllBlocks() ).toHaveLength( 1 );
+	} );
+
+	it( 'renders without crashing', async () => {
+		await expect( page ).toRenderBlock( block );
+	} );
+
+	it( 'can change title label in editor', async () => {
+		await expect( page ).toFill(
+			'.wc-block-active-filters__title > textarea',
```

Comment:
Your tests fail on the CI because this selector [is not found](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4716/checks?check_run_id=3568636698#step:11:66). I've looked in the DOM and there was no `.wc-block-active-filters__title` element.  I think you should try with another selector.

Reply:
To close the loop, the CI tests failed, as I created a PR to add the `.wc-block-active-filters__title` class. That PR had been merged accidentally last week and then reverted, but remained closed. I'll have to create a fresh PR for adding that CSS class.